apps/dashboard/src/components/primitives/badge.tsx

```diff
@@ -14,24 +14,29 @@ const badgeVariants = cva(
         soft: 'bg-neutral-alpha-200 text-foreground-500 border-transparent',
         outline: 'border-neutral-alpha-200 bg-transparent font-normal text-foreground-600 shadow-sm',
       },
-      size: {
+      kind: {
```

Comment:
renamed

Reply:
What's the default between variants and kinds?

Reply:
☝️ Same question by me. The naming feels a bit overlapping.

Reply:
size is better here as it affects paddings.

Reply:
maybe we should move the paddings to the size